src/lib/sql/sql_translator.cpp

```diff
@@ -2050,14 +2069,14 @@ SQLTranslator::TableSourceState::TableSourceState(
 
 void SQLTranslator::TableSourceState::append(TableSourceState&& rhs) {
   for (auto& table_name_and_elements : rhs.elements_by_table_name) {
-    const auto unique = elements_by_table_name.count(table_name_and_elements.first) == 0;
+    const auto unique = !elements_by_table_name.contains(table_name_and_elements.first);
     AssertInput(unique, "Table Name '"s + table_name_and_elements.first + "' in FROM clause is not unique");
   }
 
   // This should be ::merge, but that is not yet supported by clang.
   // elements_by_table_name.merge(std::move(rhs.elements_by_table_name));
-  for (auto& kv : rhs.elements_by_table_name) {
-    elements_by_table_name.try_emplace(kv.first, std::move(kv.second));
+  for (auto& element : rhs.elements_by_table_name) {
+    elements_by_table_name.try_emplace(element.first, std::move(element.second));
```

Comment:
Dues clang support `::merge` now? Or can we use `auto& [name, element] : rhs.elements_by_table_name`?